OutOfSchool/OutOfSchool.WebApi.Tests/Services/DirectionSensitiveServiceTests.cs

```diff
@@ -0,0 +1,175 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using AutoMapper;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Localization;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using OutOfSchool.Services;
+using OutOfSchool.Services.Models;
+using OutOfSchool.Services.Models.SubordinationStructure;
+using OutOfSchool.Services.Repository;
+using OutOfSchool.Tests.Common;
+using OutOfSchool.WebApi.Models;
+using OutOfSchool.WebApi.Services;
+using OutOfSchool.WebApi.Util;
+
+namespace OutOfSchool.WebApi.Tests.Services;
+
+[TestFixture]
+public class DirectionSensitiveServiceTests
+{
+    private static Guid institutionId;
+    private static Guid institutionHierarchyId;
+    private DbContextOptions<OutOfSchoolDbContext> options;
+    private OutOfSchoolDbContext context;
+    private IEntityRepositorySoftDeleted<long, Direction> repo;
+    private IWorkshopRepository repositoryWorkshop;
+    private ISensitiveDirectionService service;
+    private Mock<IStringLocalizer<SharedResource>> localizer;
+    private Mock<ILogger<DirectionService>> logger;
+    private Mock<IMapper> mapper;
+    private Mock<ICurrentUserService> currentUserServiceMock;
+    private Mock<IMinistryAdminService> ministryAdminServiceMock;
+    private Mock<IRegionAdminService> regionAdminServiceMock;
```

Comment:
(Excessive code) From all fields listed here only next (less than a half) are necessary. 
```
    private static Guid institutionId;
    private static Guid institutionHierarchyId;
    private IEntityRepositorySoftDeleted<long, Direction> repo;
    private ISensitiveDirectionService service;
    private Mock<IMapper> mapper;
```

Other could be made local variables/inlined/passed as a parameter.